components/dashboard/src/Analytics.tsx

```diff
@@ -162,18 +221,30 @@ export const trackLocation = async (includePII: boolean) => {
         url: window.location.href,
     };
 
-    getGitpodService().server.trackLocation({
-        //if the user is authenticated, let server determine the id. else, pass anonymousId explicitly.
-        includePII: includePII,
-        anonymousId: getAnonymousId(),
-        properties: props,
+    fetch("/_analytics/trackLocation", {
+        method: "POST",
+        headers: {
+            "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+            //if the user is authenticated, let server determine the id. else, pass anonymousId explicitly.
```

Comment:
Is this truy? if we use fetch, wasn't there an option we have to set to make it send the cookie?